src/components/storyUtils/Breakpoints/Breakpoints.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import useBreakpoints, { queriesKeys } from 'hooks/useBreakpoints';
+
+const MyComponent = () => {
```

Comment:
Now I am just being annoying, I know, but could you change this to Breakpoints showcase for consistency?